frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -156,28 +157,28 @@ class Sidebar extends Component<Props> {
     } = this.props;
     const hidden = ['forceHidden', 'forceCollapsed'].includes(sidebarStatus);
     return (
-      <div className={['sidebarContainer', hidden ? 'forceHide' : ''].join(' ')}>
-        <div className={['sidebarOverlay', activeSidebar ? 'active' : ''].join(' ')} onClick={toggleSidebar}></div>
-        <nav className={['sidebar', activeSidebar ? 'forceShow' : '', shown ? 'withGuide' : ''].join(' ')}>
-          <div className="sidebarLogoContainer">
+      <div className={[style.sidebarContainer, hidden ? style.forceHide : ''].join(' ')}>
+        <div className={[style.sidebarOverlay, activeSidebar ? style.active : ''].join(' ')} onClick={toggleSidebar}></div>
+        <nav className={[style.sidebar, activeSidebar ? style.forceShow : '', shown ? style.withGuide : ''].join(' ')}>
+          <div className={style.sidebarLogoContainer}>
             <Link
               to={accounts.length ? '/account-summary' : '/'}
               onClick={this.handleSidebarItemClick}>
-              <AppLogoInverted className="sidebarLogo" />
+              <AppLogoInverted className={style.sidebarLogo} />
             </Link>
-            <button className="closeButton" onClick={toggleSidebar}>
+            <button className={style.closeButton} onClick={toggleSidebar}>
               <CloseXWhite />
             </button>
           </div>
 
-          <div className="sidebarHeaderContainer">
-            <span className="sidebarHeader" hidden={!keystores?.length}>
+          <div className={style.sidebarHeaderContainer}>
+            <span className={style.sidebarHeader} hidden={!keystores?.length}>
               {t('sidebar.accounts')}
             </span>
           </div>
           {/* <div className="activeGroup">
```

Comment:
feel free to remove this whole comment, the commented code looks very much out of date.